src/Constants.mac.cs.in

```diff
@@ -146,6 +146,6 @@ namespace MonoMac {
 		public const string NaturalLanguageLibrary = "/System/Library/Frameworks/Frameworks/NaturalLanguage.framework/NaturalLanguage";
 		public const string VideoSubscriberAccountLibrary = "/System/Library/Frameworks/VideoSubscriberAccount.framework/VideoSubscriberAccount";
 		public const string UserNotificationsLibrary = "/System/Library/Frameworks/UserNotifications.framework/UserNotifications";
-		public const string iTunesLibraryLibrary = "/System/Library/Frameworks/iTunesLibrary.framewor/iTunesLibrary";
+		public const string iTunesLibraryLibrary = "/System/Library/Frameworks/iTunesLibrary.framework/iTunesLibrary";
```

Comment:
nice catch, we should add a test to ensure those file exists -> https://github.com/xamarin/xamarin-macios/issues/5047

That likely worked because the value is only used for fields, not for the bindings - and once loaded (by other means) even the fields would work properly.

Reply:
🤦‍♂️